Form-Controls/styles.css

```
@@ -0,0 +1,70 @@
+{
+    margin:0;
+    padding:0
+}
+.body{
```

```suggestion
body{
```

Typo? (install Prettier!)